src/sdk/analytics/index.tsx

```diff
@@ -1,7 +1,7 @@
 import type { AnalyticsEvent } from '@faststore/sdk'
 import { useAnalyticsEvent } from '@faststore/sdk'
 
-import storeConfig from '../../../store.config'
+import sendEvent from './platform/vtex'
```

Comment:
Is this possible? This way, as we evolve faststore everything that depends on the platform property will be automatically updated if the property changes.

```suggestion
import storeConfig from '../../../store.config'

import sendEvent from `./platform/${storeConfig.platform}`
```

Reply:
good try, but it's not possible.